src/sqlite_db.cpp

```diff
@@ -349,12 +349,13 @@ std::string SQLite::Statement::ColumnText(int idx)
         reinterpret_cast<const char*>(sqlite3_column_text(pImpl->ptrStmt.get(), idx)), bytes};
 }
 
-std::string SQLite::Statement::ColumnBlob(int idx)
+std::vector<char> SQLite::Statement::ColumnBlob(int idx)
 {
-    auto ptr = sqlite3_column_blob(pImpl->ptrStmt.get(), idx);
+    auto ptr = reinterpret_cast<const char*>(sqlite3_column_blob(pImpl->ptrStmt.get(), idx));
```

Comment:
```suggestion
    auto ptr = static_cast<const char*>(sqlite3_column_blob(pImpl->ptrStmt.get(), idx));
```
void* -> any* can be casted directly, without reinterpretation.